ang/civicase/Utils.js

```diff
@@ -154,18 +163,50 @@
       });
       return item;
     };
+
+    /**
+     * To count overdue tasks.
+     *
+     * @param {Array} activities - Array of related activities to a case
+     */
+    function countOverdueTasks (item) {
+      var ifDateInPast, isIncompleteTask, category;
+
+      item.category_count.overdue = {};
+      _.each(item.allActivities, function (val, key) {
+        category = CRM.civicase.activityTypes[val.activity_type_id].grouping || 'unlisted';
+
+        if (category) {
+          ifDateInPast = moment(val.activity_date_time).isBefore(moment());
+          isIncompleteTask = CRM.civicase.activityStatusTypes.incomplete.indexOf(parseInt(val.status_id, 10)) > -1;
+
+          if (ifDateInPast && isIncompleteTask) {
+            item.category_count.overdue[category] = item.category_count.overdue[category] + 1 || 1;
+
+            if (category !== 'communication' && category !== 'task') {
```

Comment:
```js
var otherCategories = ['communication', 'task']; // defined outside of the foreach of course

if (!_.includes(otherCategories, category)) {

}
```

this will help to 1 - easily understand which ones are the "other" categories and 2 - easily add or remove categories from this group.